record-editor/src/app/core/services/record-cleanup.service.ts

```diff
@@ -61,4 +67,20 @@ export class RecordCleanupService {
     // boolean, number
     return false;
   }
+
+  public isURL(value: any): boolean {
+    if (typeof value !== 'string') {
+      return false;
+    }
+    try {
+      const url = new URL(value);
```

Comment:
As I mentioned previously, I don't think a naive approach based only on the value and not taking into account the schema `format` (or other information) can possibly work. 

How can you distinguish a `public_notes.value` containing `https://example.com/foo contains additional information`, from a `documents.url` containing `https://example.com/file name with a space.pdf`?